pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### What issue is this solving?
+Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.
+
+### Any helpful knowledge/context for the reviewer?
```

Comment:
I like this structure! I go back and forth on if we should make the second heading "Any helpful knowledge/context for the reviewer?" a bit more pointed to help get more specific feedback . . . but the worry I have there is that getting too specific can confuse folks.

My hunch is to keep it simple like this for now and later we can get more specific if we feel we need to.